pkg/virt-launcher/virtwrap/network/podinterface.go

```diff
@@ -478,7 +478,7 @@ func (b *BridgePodInterface) decorateConfig() error {
 	ifaces := b.domain.Spec.Devices.Interfaces
 	for i, iface := range ifaces {
 		if iface.Alias.Name == b.iface.Name {
-			ifaces[i].MTU = b.virtIface.MTU
+			//ifaces[i].MTU = b.virtIface.MTU
```

Comment:
Assuming configuring the MTU requires NET_ADMIN, I've just commented this out.

There's a bug on netlink that doesn't allow the MTU for tap devices to be configured. I'm trying to address that in a netlink PR (couldn't get it working yet).